archive/frames/filters.py

```diff
@@ -5,18 +5,35 @@
 
 
 class FrameFilter(django_filters.FilterSet):
-    start = django_filters.DateTimeFilter(field_name='DATE_OBS', lookup_expr='gte')
-    end = django_filters.DateTimeFilter(field_name='DATE_OBS', lookup_expr='lte')
-    dayobs = django_filters.DateFilter(field_name='DAY_OBS', lookup_expr='iexact')
+    start = django_filters.DateTimeFilter(field_name='observation_date', lookup_expr='gte')
+    end = django_filters.DateTimeFilter(field_name='observation_date', lookup_expr='lte')
+    dayobs = django_filters.DateFilter(field_name='observation_day', lookup_expr='iexact')
     basename = django_filters.CharFilter(field_name='basename', lookup_expr='icontains')
-    OBJECT = django_filters.CharFilter(field_name='OBJECT', lookup_expr='icontains')
+    OBJECT = django_filters.CharFilter(field_name='target_name', lookup_expr='icontains')
+    target_name = django_filters.CharFilter(field_name='target_name', lookup_expr='icontains')
     public = django_filters.BooleanFilter(field_name='public', method='public_filter')
-    EXPTIME = django_filters.NumberFilter(field_name='EXPTIME', lookup_expr='gte')
+    EXPTIME = django_filters.NumberFilter(field_name='exposure_time', lookup_expr='gte')
+    exposure_time = django_filters.NumberFilter(field_name='exposure_time', lookup_expr='gte')
+    OBSTYPE = django_filters.CharFilter(field_name='configuration_type', lookup_expr='icontains')
```

Comment:
For these old filters that we'll have in here while client apps switch over to the new filters, unless I am mistaken I think they are missing `DATE_OBS`, `DAY_OBS`, and `L1PUBDAT`.

Reply:
Yeah, I'll add those in I guess. I assumed the start/end/public filters covered the date stuff but I see they could've queried by these exact values so might as well let them keep doing that for a while...